base_edi/__manifest__.py

```diff
@@ -5,8 +5,8 @@
 {
     "name": "Base EDI",
     "summary": """Base module to aggregate EDI features.""",
-    "version": "13.0.1.0.2",
-    "development_status": "Alpha",
+    "version": "13.0.2.0.0",
```

Comment:
IMO all these changes do not deserve a major release as they are backward compat...

Reply:
True, changed